### PR TITLE
[Driver] - jobFinished method now emits to stderr

### PR DESCRIPTION
**Pull request by JhonnyBillM**
Update `jobFinished(job:result:pid:)` method to emit output to `stderr` instead of `stdout`.

This reflects the implementation of the current Driver as we can see [here](https://github.com/apple/swift/blob/d930599394b27fcac77a90b63e23372c6b31cd7a/lib/Driver/Compilation.cpp#L594)

### CODE REVIEW

File: Sources/SwiftDriver/Driver/ToolExecutionDelegate.swift [hunk 1]
```diff
@@ -51,11 +51,10 @@ public struct ToolExecutionDelegate: JobExecutorDelegate {
   public func jobFinished(job: Job, result: ProcessResult, pid: Int) {
     switch mode {
     case .regular, .verbose:
-      // FIXME: Check what the current driver does.
       let output = (try? result.utf8Output() + result.utf8stderrOutput()) ?? ""
       if !output.isEmpty {
-        stdoutStream <<< output
-        stdoutStream.flush()
+				stderrStream <<< output
```
Comment (harlanhaskins) [inline Sources/SwiftDriver/Driver/ToolExecutionDelegate.swift]: Nitpick: indentation

Comment (JhonnyBillM) [reply Sources/SwiftDriver/Driver/ToolExecutionDelegate.swift]: Fixed.

Sorry, didn't see that, seems like I changed my Xcode indentation settings.

**Comment by harlanhaskins** [comment]
@swift-ci please test

**Comment by harlanhaskins** [comment]
Thanks @JhonnyBillM! Welcome to the Swift project!

**Pull request merged**